### ISSUE TITLE
Can't get it working on macOs BigSur with PHP 8

### ISSUE THREAD

**Issue by thinkstudeo** [beyondcode/laravel-debugbar-companion#6]
macOs version: 11.1
php version: 8.0
companion app version: 0.1.0
Laravel Valet 2.13.16

Steps I followed:

- updated barryvdh/laravel-debugbar to v 3.5.2
- vendor:publish the config file
- downloaded the `.dmg` from releases page & moved it to applications

Modified the `config/debugbar.php` as
```php
'storage' => [
        'enabled'    => true,
        'driver'     => 'socket', // redis, file, pdo, socket, custom
        'path'       => storage_path('debugbar'), // For file driver
        'connection' => null,   // Leave null for default connection (Redis/PDO)
        'provider'   => '', // Instance of StorageInterface for custom driver
        'hostname'   => '127.0.0.1', // Hostname to use with the "socket" driver
        'port'       => 2304, // Port to use with the "socket" driver
    ],
```

But when I open the app it shows 
<img width="1008" alt="Screenshot 2021-01-06 at 02 49 56" src="https://user-images.githubusercontent.com/29278334/104057953-f7803780-5218-11eb-8097-30fd871342e5.png">

What am I missing? Why doesn't it work?

**Comment by csoutham** [comment]
@thinkstudeo I was having the same issue - as the instructions say to use driver: app but I've just seen you have driver: socket. I've switched to this and it's now working for me. I'm running Valet and PHP v7.4.
However I can see you've already got that set in your debugbar config.

Is DebugBar already running on your front end?

**Comment by thinkstudeo** [comment]
> Is DebugBar already running on your front end?

@csoutham Yes it is. And I tried switching to PHPv7.4.13 (via brew) but still the same

Laravel Valet 2.13.16

**Comment by leandrodiogenes** [comment]
Same here
PHP 7.4.5
Windows 10
Laravel 8.12

![image](https://user-images.githubusercontent.com/2463299/104130096-a02fb380-534d-11eb-9f5d-b5f35f35a88f.png)
![image](https://user-images.githubusercontent.com/2463299/104130102-a9b91b80-534d-11eb-82c0-b0b396bce30e.png)
![image](https://user-images.githubusercontent.com/2463299/104130105-afaefc80-534d-11eb-8222-16dcbec9359c.png)

**Comment by DavyDeCoster** [comment]
@leandrodiogenes You can try changing the driver-setting in `config/debugbar.php` to `socket`. The 0.1 release doesn't have the latest information.

**Comment by sebastiaanluca** [comment]
Can't get it to work either. Did some debugging and it sends the encoded payload to the socket, but the app doesn't do much 😄

Debugbar v3.5.2
Laravel Framework 8.20.1
PHP 7.4.13
macOS 10.15.7

![Screen Shot 2021-01-12 at 6 53 47 PM](https://user-images.githubusercontent.com/711940/104352896-89cd5780-5507-11eb-8770-dffc6a120e05.JPG)

**Comment by DavyDeCoster** [comment]
@sebastiaanluca Can you look in the dev-tools of the app and see if there's a error in the console? Could be a problem which is fixed already, but isn't released yet.

**Comment by sebastiaanluca** [comment]
@DavyDeCoster Ah yes!

```
chunk-vendors.51d78ec2.js:6 TypeError: Cannot read property 'count' of undefined
    at o (app.dbed4ebf.js:1)
    at o.Tn.e._render (chunk-vendors.51d78ec2.js:6)
    at o.r (chunk-vendors.51d78ec2.js:6)
    at rr.get (chunk-vendors.51d78ec2.js:6)
    at rr.run (chunk-vendors.51d78ec2.js:6)
    at Xn (chunk-vendors.51d78ec2.js:6)
    at Array.<anonymous> (chunk-vendors.51d78ec2.js:6)
    at _t (chunk-vendors.51d78ec2.js:6)
```

***

Noticed https://github.com/beyondcode/laravel-debugbar-companion/issues/7#issuecomment-757999274 (same issue). Will try to build the master, thanks!

**Comment by thinkstudeo** [comment]
Building it from the master does solve the issue. Thanks @DavyDeCoster @sebastiaanluca for pointing the direction
